ExpandableView/ExpandableView.cs

```diff
@@ -266,13 +266,17 @@ private View CreateSecondaryView()
 
         private void OnSecondaryViewSizeChanged(object sender, EventArgs e)
         {
-            if (SecondaryView.Height <= 0) return;
+            if (SecondaryView.Height <= 0 || SizeChangeIsInsignificant()) return;
```

Comment:
why OR ?

Reply:
Because you don't want to animate insignificant size changes, specifically those caused by floating point arithmetic errors.